.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
The mentioned pattern could be used here as well, for example:

```
## Description of the problem

<!--
Is your feature request related to a problem? Please describe.
A clear and concise description of what the problem is. Ex. I'm always frustrated when
-->

## Description of the feature

<!--  A clear and concise description of what you want to happen.  -->

## Alternatives

## Additional context

<!-- Add technical implementation details, screenshots, any other context. -->
```

Reply:
Thanks, done! :)